packages/custom-webpack/examples/full-cycle-app/angular.json

```diff
@@ -18,9 +18,12 @@
           "builder": "@angular-builders/custom-webpack:browser",
           "options": {
             "customWebpackConfig": {
-              "path": "./extra-webpack.config.ts"
+              "path": "extra-webpack.config.ts",
+              "mergeStrategies": {
```

Comment:
[`mergeRules`](https://github.com/just-jeb/angular-builders/blob/master/MIGRATION.MD#migration-from-version-10-to-version-11) should be used instead